### PR TITLE
Add contact details to sponsor

### PR DESCRIPTION
**Pull request by margOnline**
[WIP] Addresses issue #229

The view needs work and has not been completed as I would like clarification on a couple of points:
* what is meant by 'ability to assign members as contacts for a sponsor'. 
  * Can it be any member?
  * I've assumed a one-to-many association between member and sponsor
* Any thoughts on the ui for selecting a member to be the contact point?
* Any requirements for validation on the new fields?

### CODE REVIEW

**Comment by despo** [comment]
Hey @margOnline 

Yes, any member sounds right. I don't think any validations are necessary since they should already have an email setup - and we don't really need a phone number.

A dropdown box I think, with multiple select enabled similar to here: 
https://github.com/codebar/planner/blob/master/app/views/admin/workshops/_edit_form.html.haml#L20

No validation on the new fields either as we definitely have existing sponsors whose contact details we dont necessarily have.

I hope this answers your questions. Let me know if there is anything else.

File: app/models/sponsor.rb [hunk 1]
```diff
@@ -3,9 +3,10 @@ class Sponsor < ActiveRecord::Base
   has_one :address
   has_many :sponsor_sessions
   has_many :sessions, through: :sponsor_sessions
+  belongs_to :member
```
Comment (despo) [inline app/models/sponsor.rb]: This should also be a many to many association, as a sponsor could have more than one points of contact.

**Comment by despo** [comment]
So, a sponsor should have a contact email, as we currently dont store that in the app which means we need to look around at times to find details.

But, a sponsor should also have many members as points of contacts. This should in the future enable us to prioritize coaches, or add them to the workshop whitelist if the event is hosted at their venue.

File: app/views/admin/sponsors/new.html.haml [hunk 2]
```diff
@@ -17,6 +17,11 @@
                 = f.input :website, as: :url
             .row
               .large-3.columns
+                = f.input :contact_first_name
+              .large-3.columns
+                = f.input :contact_surname
+            .row
+              .large-3.columns
```
Comment (despo) [inline app/views/admin/sponsors/new.html.haml]: This should also include an email address. (Our sponsors dont need to be codebar members)

**Comment by deniseyu** [comment]
@margOnline happy to review this for merge, please pull latest changes from upstream and resolve conflicts. Thanks for your work :smile:

**Pull request merged**